test/functional/fixtures/api/es-next/click/test.js

```diff
@@ -123,4 +123,8 @@ describe('[API] t.click()', function () {
                 });
         });
     });
+
```

Comment:
My fault, i've placed them correctly for now.
I've missed some details in issue description, so i changed it to be clear, what we fix.